source/CardGames.py

```diff
@@ -161,3 +175,35 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+# simple respond for dealer in single player with known hands 
+def dealerHand(dealer, house, player):
+  houseHand = house.handSum()
+  playerHand = player.handSum()
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1   
+  if houseHand == 21 and len(house.hand) == 2: #if blackjack then house auto win
+    return -1 
+  while houseHand < 17: #force hit when below 17
+    dealer.dealCards(1, [house])
+    houseHand = house.handSum()
+
+  if playerHand <= 21: 
+    while playerHand >= houseHand: # keep hitting until house hand value is bigger than player hand, unless houseHand reach 21
```

Comment:
The house does not respond to the player. The house operates on a strict algorithm where it must hit if their hand is <17 else they stand if their handsum is >= 17.